packages/flutter_tools/test/commands.shard/permeable/build_appbundle_test.dart

```diff
@@ -97,64 +96,15 @@ void main() {
 
   group('Gradle', () {
     Directory tempDir;
-    ProcessManager mockProcessManager;
-    MockAndroidSdk mockAndroidSdk;
-    String gradlew;
+    FakeProcessManager processManager;
+    FakeAndroidSdk mockAndroidSdk;
     TestUsage testUsage;
 
     setUp(() {
       testUsage = TestUsage();
       tempDir = globals.fs.systemTempDirectory.createTempSync('flutter_tools_packages_test.');
-      gradlew = globals.fs.path.join(tempDir.path, 'flutter_project', 'android',
```

Comment:
There were no verify calls or anything, so the test doesn't really depend on this mock at all. Switched to .any and everything still passes